libs-scala/resources/src/main/scala/com/digitalasset/resources/ProgramResource.scala

```diff
@@ -46,7 +46,12 @@ class ProgramResource[T](
               stop()
               sys.exit(1)
           }
-        case Failure(_: SuppressedException) =>
+        case Failure(exception: StartupException) =>
+          logger.error(
+            s"Shutting down because of an initialization error.\n${exception.getMessage}")
```

Comment:
```suggestion
            s"Shutting down because of an initialization error: ${exception.getMessage}", exception)
```
Why the newline? It may be just escaped if structured logging is used, anyhow.

Reply:
It's not, I checked. The error looks like this:

```
Shutting down because of an initialization error.
Unknown database: jdbc:notathing
```

If I pass the exception as an argument, I get this:

```
Shutting down because of an initialization error.
com.daml.ledger.on.sql.Database$InvalidDatabaseException: Unknown database: jdbc:notathing
```